robot-tests/collect_screenshots.py

```
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+"""
+Script to collect and organize screenshots from test results
+"""
+
+import os
+import shutil
+import glob
```

Ruff (0.12.2)

`glob` imported but unused

Remove unused import: `glob`

(F401)